docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Is there a way we can remove the hardcoded <code>ATL08_20230816182927_08792008_006_01.h5</code> by a filename created using date or something else?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I probably could! However, I may leave it alone as I think that's how they're kept in other tutorials, and also how they show up in NASA CMR 🙂</p>